src/python-common/ceph/deployment/drive_group.py

```diff
@@ -16,20 +16,27 @@ class DeviceSelection(object):
     specification structure.
     """
 
-    def __init__(self, paths=None, id_model=None, size=None, rotates=None, count=None):
-        # type: (List[str], str, str, bool, int) -> None
+    def __init__(self,
+                 paths=None,  # type: Optional[List[str]]
+                 id_model=None,  # type: Optional[str]
+                 size=None,  # type: Optional[str]
+                 rotates=None,  # type: Optional[bool]
+                 count=None,  # type: Optional[int]
+                 vendor=None,  # type: Optional[str]
+                 all_devices=False  # type: bool
+                 ):
         """
         ephemeral drive group device specification
         """
-        if paths is None:
-            paths = []
-
         #: List of absolute paths to the devices.
-        self.paths = paths  # type: List[str]
+        self.paths = [] if paths is None else paths  # type: List[str]
 
         #: A wildcard string. e.g: "SDD*"
         self.id_model = id_model
 
+        #: Match on the VENDOR property of the drive
+        self.vendor = vendor
+
         #: Size specification of format LOW:HIGH.
         #: Can also take the the form :HIGH, LOW:
         #: or an exact value (as ceph-volume inventory reports)
```

Comment:
Can we also make comment for `size` clearer? For example, to support the same flavor as the drive group in DeepSea. This not only affects how orchestrator backend implements related features but also affects how clients (like Dashboard) pass the value. Thanks!